elasticdl/python/ps/servicer.py

```diff
@@ -249,3 +244,12 @@ def _save_params_to_checkpoint_if_needed(self):
                 shard_index=self._ps_id,
                 shard_num=self._num_ps_pods,
             )
+
+    def _set_optimizer_learning_rate(self, learning_rate):
+        if learning_rate is None:
+            return
+
+        if self._use_wrap_opt:
+            self._optimizer.set_learning_rate(learning_rate)
```

Comment:
```python
def set_learning_rate(self, learning_rate):
        K.set_value(self._opt.lr, K.get_value(learning_rate))
```

It seems that there is no need to add a `if/else` here.

Reply:
The ` self._optimizer` maybe an `OptimizerWrapper` instance or `Optimizer` instance, so it needs `if/else`.